lib/wallet/nodeclient.js

```diff
@@ -211,6 +211,20 @@ class NodeClient extends AsyncEmitter {
       return this.emitAsync('block rescan', entry, txs);
     });
   }
+
+  /**
+  * Get relevant node info for wallet
+  * @returns {Promise}
```

Comment:
Why is getInfo() async? Seems redundant, given that it returns a resolved promise and it's `await`ed for in all the places where it's called.

Reply:
Thanks Orfeas, just cleaned this up.